### PR TITLE
Cron to update catalog #3469

### PR DESCRIPTION
**Pull request by priyankarani**
Review ID: 1623003

### CODE REVIEW

**Comment by sharoonthomas** [comment]
@shalabhaggarwal do you agree with this patch ?

File: .travis.yml [hunk 1]
```diff
@@ -13,6 +13,7 @@ install:
   - pip install "pyparsing==1.5.7"
   - pip install flake8
   - ./install_nightly.sh
+  - pip install distribute
```
Comment (shalabhaggarwal) [inline .travis.yml:16]: I dont think this is needed.

Even if needed, should go as a separate commit

**Comment by shalabhaggarwal** [comment]
The patch is fine.

I feel the docstring is a bit confusing and should say that this update happens from magento to openerp.

Also IMO the cron should be active = False by default.
